CameraExtrinsicMat:
   rows: 4
   cols: 4
   dt: d
   data: [-0.0468903 ,  0.0452459 ,  0.997875  , -0.11109102, 
          -0.998525  ,  -0.0294874 , -0.0455838 ,  0.11441981,  
           0.0273622 , -0.998541   , 0.0465618 , -0.07174107,
                0.,  0.,  0.,  1.]
CameraMat:
   rows: 3
   cols: 3
   dt: d
   data: [ 1.927941866078295e+03, 0., 9.595316580313911e+02,
           0.,  1.929172443177110e+03, 5.144334699977888e+02, 
           0.,  0.,                    1. ]
DistCoeff:
   rows: 1
   cols: 5
   dt: d
   data: [  -0.2272996302077433, 0.244566784018986, 4.524726839204985e-05, -0.0008645881261561915,  -0.2066175572621939]
ImageSize: [ 1920, 1080 ]
ReprojectionError: 0
DistModel: plumb_bob
